.github/workflows/build-and-deploy.yml

```diff
@@ -1,19 +1,60 @@
 name: Build and deploy on push
 
 on:
+  workflow_dispatch:
   push:
     branches:
+      - dev
```

Comment:
Change dev to development to match the current repo.
Also change main to master. Ill change it when i  rename everything to main.